docs/source/index.rst

```diff
@@ -0,0 +1,107 @@
+.. PyDESeq2 documentation master file, created by
+   sphinx-quickstart on Thu Dec 22 16:34:24 2022.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to PyDESeq2's documentation!
+====================================
+
+This package is a python implementation of the `DESeq2 <https://bioconductor.org/packages/release/bioc/html/DESeq2.html>`_ method [1]_
+for differential expression analysis (DEA) with bulk RNA-seq data, originally in R.
+It aims to facilitate DEA experiments for python users.
+
+As PyDESeq2 is a re-implementation of `DESeq2 <https://bioconductor.org/packages/release/bioc/html/DESeq2.html>`_ from scratch, you may experience some differences in terms of retrieved values or available features.
+
+Currently, available features broadly correspond to the default settings of DESeq2 (v1.34.0) for single-factor analysis,
```

Comment:
Adding a citation to apeGLM:

```suggestion
Currently, available features broadly correspond to the default settings of DESeq2 (v1.34.0) for single-factor analysis with an optional `apeGLM <https://academic.oup.com/bioinformatics/article/35/12/2084/5159452>`_ LFC shrinkage [2]_,
```

Reply:
good idea. I add this sentence and the associated reference